src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,31 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String FIRST_CHAR = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> lines;
+        List<String> list = new ArrayList<>();
+
+        try {
+            lines = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read this file", e);
+        }
+
+        for (String s : lines) {
```

Comment:
please set informative names for your variables
```suggestion
        for (String line : lines) {
```

Reply:
done